app/ui-utils/client/lib/RoomManager.js

```diff
@@ -265,34 +248,29 @@ export const RoomManager = new function() {
 			return onlineUsers.set(onlineUsersValue);
 		}
 
-		updateMentionsMarksOfRoom(typeName) {
-			const dom = this.getDomOfRoom(typeName);
-			if (!dom) {
-				return;
-			}
-
-			const [ticksBar] = dom.getElementsByClassName('ticks-bar');
-			const [messagesBox] = dom.getElementsByClassName('messages-box');
-			const scrollTop = $('> .wrapper', messagesBox).scrollTop() - 50;
-			const totalHeight = $(' > .wrapper > ul', messagesBox).height() + 40;
-			if (!ticksBar) {
-				return;
-			}
-
-			// TODO: thread quotes should NOT have mention links at all
-			const mentionsSelector = '.message .body .mention-link--me, .message .body .mention-link--group';
-			ticksBar.innerHTML = Array.from(messagesBox?.querySelectorAll(mentionsSelector) || [])
-				.map((mentionLink) => {
-					const topOffset = $(mentionLink).offset().top + scrollTop;
-					const percent = (100 / totalHeight) * topOffset;
-					const className = [
-						'tick',
-						mentionLink.classList.contains('mention-link--me') && 'tick--me',
-						mentionLink.classList.contains('mention-link--group') && 'tick--group',
-					].filter(Boolean).join(' ');
-					return `<div class="${ className }" style="top: ${ percent }%;"></div>`;
-				})
-				.join('');
+		updateMentionsMarksOfRoom(/* typeName */) {
```

Comment:
Can you remove that? Or is something important to stay here like a comment?

Reply:
actually is a feature removed by now, but I want to restore soon as possible